src/ScreenCover/index.js

```diff
@@ -3,14 +3,16 @@ import propTypes from 'prop-types'
 import css from 'styled-jsx/css'
 import cx from 'classnames'
 
-const Backdrop = ({ onClick }) => (
+const Backdrop = ({ onClick, withoutBackgroundColor }) => (
```

Comment:
I don't understand why we need a backdrop when creating the menus? It is not legal according to the stacking context in the design system: https://github.com/dhis2/design-system/blob/master/principles/spacing-alignment.md#stacking

The Popup components belong on the _Appliation top layer_ with a `z-index` of `2000`. The `ScreenCover` is a _Blocking layer_ which belongs on `z-index` of `3000`.

_If necessary_: rename `withoutBackgroundColor` to `transparent` and set the background to `transparent` instead of `none`.